src/components/Game/Backspace/Backspace.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classes from './Backspace.scss';
 
-const Backspace = ({ clicked, areSlotsEmpty }) => (
+const Backspace = ({ clicked, areSlotsEmpty, isSubmitVisible }) => (
     <button
         onClick={clicked}
         className={classes.BackspaceBtn}
-        disabled={areSlotsEmpty}
+        disabled={areSlotsEmpty || isSubmitVisible}
```

Comment:
it would be cleaner to pass `disabled` prop because it's more elegant if a component is not troubled with logic which is unrelated to what this components represents
`Backspace` represents an element of UI for deleting something, that's all

passing `isSubmitVisible` to `Backspace` is a bit like telling a clerk at your local bakery how you make your tea in the morning, a bit unrelated ;)

ofc it can stay like it is and no panda will die but I feel my responsibility in this project is to voice my opinion so I'm doing just that!

Reply:
👍